<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>308</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>293</width>
    <height>308</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(56, 58, 89);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="le_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>102</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);

}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  EMAIL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_clave">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>162</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solidrgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);
}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>  PASSWORD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_login">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 8px;
	background-color: rgb(185, 212, 248);
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	
	background-color: rgb(101, 155, 219);
}</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/Icon USer.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_crear_cuenta">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>278</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Crear una cuenta</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_error">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
